python/cuml/internals/base_helpers.py

```diff
@@ -103,8 +104,7 @@ def _wrap_attribute(class_name: str,
                     **kwargs):
 
     # Skip items marked with autowrap_ignore
-    if ("__cuml_is_wrapped" in attribute.__dict__
-            and attribute.__dict__["__cuml_is_wrapped"]):
+    if (attribute.__dict__.get("__cuml_is_wrapped", False)):
```

Comment:
Nitpick: _deprecate_pos_args.FLAG_NAME is always used through a constant and this flag, which seems conceptually almost the same, is used through a literal

Reply:
I couldnt do exactly the same thing (There is no common decorator class for `__cuml_is_wrapped` to be attached to) but I did create the constant `CUML_WRAPPED_FLAG`. Let me know if this resolves this

Reply:
Yeah, looks good to me now.